frontend/controller/actions/identity.js

```diff
@@ -215,9 +218,13 @@ export default (sbp('sbp/selectors/register', {
   'gi.actions/identity/logout': async function () {
     const state = sbp('state/vuex/state')
     try {
+      // wait for any pending sync operations to finish before saving
+      console.info('logging out, waiting for any events to finish...')
```

Comment:
I'd rather use console.log by default, and keep console.info only to display things related to the runtime environment like current app/library versions, active user settings, detected browser/platform features or public environment variables.

Reply:
I wanted to leave it as `.info` here so that we can debug issues with any potential logouts that hang. Unlikely to happen but not impossible with the `/wait` below